pkg/controllers/syncer/syncer.go

```diff
@@ -217,12 +218,19 @@ func (r *syncerController) enqueuePhysical(obj client.Object, q workqueue.RateLi
 		klog.Errorf("error checking object %v if managed: %v", obj, err)
 		return
 	} else if !managed {
+		// klog.Infof("returning without enqueuing since not managed")
 		return
 	}
 
 	name := r.syncer.PhysicalToVirtual(obj)
 	if name.Name != "" {
 		q.Add(reconcile.Request{NamespacedName: name})
+	} else if r.options.IsClusterScopedCRD {
+		q.Add(reconcile.Request{
+			NamespacedName: types.NamespacedName{
+				Name: obj.GetName(),
+			},
+		})
```

Comment:
Why is this branch needed? When would the `name.Name` be empty?
Also, considering how import syncer implements `PhysicalToVirtual()` when `r.options.IsClusterScopedCRD` is true, the outcome would be the same as in the branch above :thinking:

Reply:
Yea, that's true, I guess I added this later on inside `PhysicalToVirtual()` and missed removing this `else if`